helm-charts/prometheus_operator_values.yaml

```diff
@@ -1853,7 +1853,7 @@ prometheus:
          resources:
            requests:
              storage: 100Gi
-       selector: {}
+#       selector: {}
```

Comment:
Just want to call this line out and make sure this was intentional since the line is commented-out and not deleted.

Reply:
Yes, Helm 3 did not like it and it seems like most not used values are commented, so I have followed along.